.cspell/computer-terms.txt

```diff
@@ -0,0 +1,354 @@
+devicetypes
+reflashing
+reflashed
+AUFS
+reprovisioning
+uuidgen
+tmpimg
+tmpconfig
+Reprovisioning
+BALENARC
+sshkeys
+tagkey
+Ijdo
+SCESU
+HKAH
+balenablocks
+forkable
+composeyml
+isnt
+Aetina
+Auvidea
+Orbitty
+Spacely
+nanopi
+preinstall
+alsa
+libasound
+gpio
+RAWIO
+udev
+Beaglebone
+sysfs
+unexport
+EEPROM
+Octalbonescript
+libmraa
+Capemgr
+MRAA
+dtparam
+digitiser
+miniuart
+variablename
+vcore
+sdram
+overscan
+capemgr
+OCPDIR
+fdtfile
+emmc
+mraa
+microcontroller
+devtmpfs
+udevd
+udevadm
+mutli
+bootloader
+sysroot
+PPPS
+uhubctl
+UDEV
+Avahi
+aufs
+Livepush
+nolive
+dockerhub
+webterminal
+printenv
+someservice
+bluez
+timedate
+rmmod
+apikey
+sandboxing
+IPAM
+dnsmasq
+myapp
+Automounting
+automounting
+devicehost
+SSID
+ssuccessful
+balenafying
+Authy
+createapikey
+rollouts
+fkms
+FBCP
+balenayml
+configtxt
+journalctl
+conntrack
+Connectivty
+effectwarning
+mitigations
+configizer
+tuneables
+Cloudlink
+updatability
+Passwordless
+preconfigured
+TOTP
+aspnet
+minideb
+INITSYSTEM
+Initsystem
+armel
+openrc
+hugepages
+logind
+xbuild
+endevour
+allwinner
+elinux
+ralink
+firmwares
+hotswap
+lirc
+balenaos
+reprovision
+modeswitch
+blox
+LBEL
+AAAAB
+macbook
+zenbook
+configjson
+backported
+Backports
+backports
+Hotspot
+ssid
+Ssid
+autoconnect
+peap
+mschapv
+eduroam
+bssids
+giffgaff
+wsec
+endianity
+dnsservers
+nmcli
+passwordless
+Dnsmasq
+resolvconf
+openembedded
+deviceip
+bblayers
+developmentmode
+tzdata
+chronyd
+minpoll
+maxpoll
+driftfile
+chronyc
+ntpdata
+ntpservers
+resinhup
+unbootable
+Undervoltage
+undervolted
+cpuset
+healthcheck
+ulimits
+userns
+blkio
+memswap
+barebones
+avahi
+mmcli
+promtail
+sdjournal
+bootable
+replug
+usbboot
+usboot
+flashall
+usbutils
+coreutils
+UEFI
+SATA
+devboard
+uefi
+Digi
+odroid
+Hardkernel
+aplus
+bplus
+Customboard
+CEST
+networkmanager
+highspeed
+roadmap
+universaln
+cacache
+persistentlogging
+MASKROM
+beagleboard
+leds
+Compulab
+Connnect
+microswitches
+nanopc
+uboot
+orangepi
+qemux
+virtio
+nographic
+revpi
+Technologic
+SDCARD
+metainfo
+hummingboard
+otherboards
+chek
+myorg
+gstatic
+balenadevice
+iname
+wrappper
+tooltipped
+selectpicker
+eoan
+retrun
+cllient
+idfa
+endmacro
+pageview
+subshell
+gsub
+prefixoid
+discoveryjs
+topo
+jridgewell
+sindresorhus
+estree
+webassemblyjs
+xtuc
+configtest
+envinfo
+amdefine
+timsort
+arrify
+autolinker
+execa
+filenamify
+pify
+isurl
+nextick
+depd
+opencollective
+caniuse
+picocolors
+keyv
+responselike
+callsites
+camelcase
+colorette
+imurmurhash
+picomatch
+universalify
+icss
+cssesc
+domutils
+decamelize
+tarbz
+targz
+unbzip
+domelementtype
+domhandler
+encodeurl
+tapable
+arrayish
+esrecurse
+estraverse
+esparse
+esvalidate
+pseudomap
+yallist
+isexe
+finalhandler
+parseurl
+extglob
+randomatic
+klaw
+striptags
+toidentifier
+wrappy
+reinterpolate
+templatesettings
+typer
+unyield
+multimatch
+iojs
+whatwg
+popperjs
+rechoir
+bunzip
+uglifyjs
+esbuild
+watchpack
+webidl
+cloudiotcore
+gcloud
+FORMATED
```

Comment:
Wow that's a big list, where did we get this from or did you make this @zoobot? 
If it's the latter, would it be a good idea to contribute this back into Cspell?